packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts

```diff
@@ -30,6 +30,11 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
   componentRef?: (component: IComboBox) => void;
 
   /**
+   * If there is a menu open this will dismiss the menu
+   */
+  dismissMenu?: () => void;

```

Comment:
This is not correct. The function should be exposed via the IComboBox interface and then you can call it from the comboBox componentRef

Reply:
This is a whole lot easier than I originally made it, thank you!